packages/synapse-react-client/src/components/JsonSchemaForm/widgets/SelectWidget.tsx

```diff
@@ -49,17 +49,28 @@ export const SelectWidget: Widget = (props: SelectWidgetProps) => {
     onBlur,
     onFocus,
     placeholder,
-    isClearable = true,
+    isClearable: _isClearable = true,
     rawErrors,
     registry,
   } = props
+
   const { enumOptions } = options as {
     enumOptions: EnumOptionsType[]
   }
-  const allowFreeSolo = Boolean(
-    (registry.formContext as CustomFormContext).allowFreeSoloEnum,
+
+  // If the ID includes one of these, then this is a widget used to select a schema in `oneOf` or `anyOf`.
+  // In these cases, we should not allow free solo input.
+  const isAnyOrOneOfOption = ['__oneof_select', '__anyof_select'].some(
```

Comment:
Are these ids stable?  Or are they exported constants that can be referenced?

Reply:
These are stable but not exported by RJSF: https://github.com/rjsf-team/react-jsonschema-form/blob/60d57d36a950a49ee801129374b25aea0319d66d/packages/core/src/components/fields/MultiSchemaField.tsx#L137-L140